README.md

```diff
@@ -2,7 +2,7 @@
 
 The Cluster Network Operator installs and upgrades the networking components on an OpenShift Kubernetes cluster.
 
-It follows the [Controller pattern](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg#hdr-Controller): it reconciles the state of the cluster against a desired configuration. The configuration specified by a CustomResourceDefinition called `networkoperator.openshift.io/NetworkConfig/v1`, which has a corresponding [type](/openshift/cluster-network-operator/blob/master/pkg/apis/networkoperator/v1/networkconfig_types.go).
+It follows the [Controller pattern](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg#hdr-Controller): it reconciles the state of the cluster against a desired configuration. The configuration specified by a CustomResourceDefinition called `networkoperator.openshift.io/NetworkConfig/v1`, which has a corresponding [type](/pkg/apis/networkoperator/v1/networkconfig_types.go).
```

Comment:
hm... that will link to the copy of the file in whatever fork/branch you're looking at. Not sure if Casey had intended for it to always link to the master copy? (In which case the fix would be to add "`https://github.com`" before it). I'll let him comment...

Reply:
GitHub URLs always confuse me. This seems fine.